examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>Finally, let's try to forecast the middle level series directly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>